test/client.js

```diff
@@ -256,10 +270,16 @@ describe('Client', () => {
         headers: mockHeaders,
         body: MOCK_BODY,
       };
-      const mockDevice = MOCK_DEVICE_TOKEN;
-      const result = await client.write(mockNotification, mockDevice);
-      expect(result).to.deep.equal({
-        device: MOCK_DEVICE_TOKEN,
+      const device = MOCK_DEVICE_TOKEN;
+      let receivedError;
+      try {
+        await client.write(mockNotification, device, 'device', 'post');
+      } catch (e) {
+        receivedError = e;
```

Comment:
Since the `write` method now can throw errors, changes to the test look like this, but still keeps the same type of results